orgbee/src/app/pages/freedom-wall/freedom-wall.component.html

```diff
@@ -1 +1,36 @@
-<p>freedom-wall works!</p>
+<div class="top">
+  <div class="header">
+    <h3>Welcome to</h3>
+    <h1>Computer Society 2024 Freedom Wall</h1>
+  </div>
+
+  <div class="button-container">
+    <button class="create-post-btn" (click)="toggleModal()">
+      <img src="../../../assets/logo.png">
+        Create an anonymous post</button>
+    <div>
+      <button class="delete-post-btn">Delete Post</button>
+      <button class="manage-wall-btn">Manage Wall</button>
+    </div>
+  </div>
+</div>
+
+<div class="post-creation-modal" *ngIf="showModal">
+  <div class="modal-content">
+    <span class="close-btn" (click)="showModal = false">&times;</span>
+    <textarea [(ngModel)]="newPostText" 
+        placeholder="Write your post here...">
+    </textarea>
+    <button (click)="addPost(); showModal = false">Add Post</button>
+  </div>
+</div>
+
+<ngx-masonry class="masonry-container">
+  <ngx-masonry-item *ngFor="let post of posts">
+    <div class="masonry-item" [ngStyle]="{'background-color': post.color}" 
```

Comment:
Never use NGstyle put it in css instead.

reference: https://docs.google.com/presentation/d/1Mdcq3WcHksBlbCvxi6wx5vajn4v8LGlJRnBIu3x4fvw/edit#slide=id.g2ce365fc0ac_0_0